q.js

```diff
@@ -1645,10 +1645,10 @@ function any(promises) {
         function onRejected(err) {
             pendingCount--;
             if (pendingCount === 0) {
-                var rejection = err || new Error("Rejection was null/undefined");
+                var rejection = err || new Error("Rejection value was: " + err);
```

Comment:
I agree it can be too verbose. My attempt was to make sure the error message could lead you to the real cause, but `Last Error: undefined` is enough & clear.